dango/genesis/src/lib.rs

```diff
@@ -250,14 +281,33 @@ where
         "dango/taxman",
     )?;
 
+    // Instantiate the oracle contract.
+    let oracle = instantiate(
+        &mut msgs,
+        oracle_code_hash,
+        &oracle::InstantiateMsg {
+            guardian_set: btree_map!(
+                GUARDIAN_INDEX => GuardianSetInfo {
+                        expiration_time: 0,
+                        addresses: GUARDIANS_ADDRESSES.into_iter().map(|addr| {
+                            Hash160::from_inner(Binary::from_str(addr)
+                            .unwrap().into_inner().try_into().unwrap())
+                        }).collect() }
+            ),
```

Comment:
this part isn't formatted

Reply:
also always use curly braces for this macro: `btree_map! { /* ... */ }` instead of `btree_map!(/* ... */)`